pandas/tests/series/accessors/test_dt_accessor.py

```diff
@@ -582,13 +582,15 @@ def test_strftime_dt64_days(self):
         # dtype may be S10 or U10 depending on python version
         tm.assert_index_equal(result, expected)
 
-    def test_strftime_period_days(self):
+    def test_strftime_period_days(self, using_infer_string):
         period_index = period_range("20150301", periods=5)
         result = period_index.strftime("%Y/%m/%d")
         expected = Index(
             ["2015/03/01", "2015/03/02", "2015/03/03", "2015/03/04", "2015/03/05"],
             dtype="=U10",
         )
+        if using_infer_string:
+            expected = expected.astype("string[pyarrow_numpy]")
```

Comment:
I forgot if/when this was discussed, but I thought `pd.ArrowDtype(pa.string())` was going to be the new default?

Reply:
No we discussed this at the summit that we would use a string dtype that follows numpy semantics with arrow backing

Reply:
@mroeschke https://github.com/pandas-dev/pandas/issues/54792 tracks that